### PR TITLE
OpenTTD update recipe

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
* Added HVIF icon using a RDef file
* Removed patches
* Update recipe to not use patches and apply the RDef file

### CODE REVIEW

**Comment by waddlesplash** [comment]
Link to GCI task please.

**Comment by AdrianArroyoCalle** [comment]
https://codein.withgoogle.com/dashboard/task-instances/4566970007552000/

File: games-simulation/openttd/additional-files/openttd.rdef [hunk 1]
```diff
@@ -0,0 +1,121 @@
+resource app_flags B_SINGLE_LAUNCH;
+
+
```
Comment (humdingerb) [inline games-simulation/openttd/additional-files/openttd.rdef]: Can you add an app signature, like   
```resource app_signature "application/x-vnd.OpenTTD";```

**Comment by AdrianArroyoCalle** [comment]
Added "app_signature"

**Comment by pulkomandy** [comment]
Is this still "WIP" or can it be merged?

**Comment by AdrianArroyoCalle** [comment]
Merge, please

**Pull request merged**